appengine/channel/app.yaml

```diff
@@ -1,8 +1,18 @@
-application: YOUR_PROJECT_ID
+# uncomment below and replace with your Project ID
+# application: YOUR_PROJECT_ID
+module: tictactoe
 version: 1
 runtime: python27
 api_version: 1
+threadsafe: true
 
 handlers:
 - url: /.*
-  script: chatactoe.py
+  script: chatactoe.app
+  #script: chatactoe.py
+
+libraries:
+- name: webapp2
+  version: latest
```

Comment:
please use the real library versions, available here: https://cloud.google.com/appengine/docs/python/tools/libraries27